File: app/src/main/java/com/google/android/apps/photos/cameraassistant/CameraAssistantService.java
package com.google.android.apps.photos.cameraassistant;

import android.app.Service;
import android.content.Intent;
import android.util.Log;
import android.os.IBinder;
import android.content.res.Configuration;

public class CameraAssistantService extends Service {
  @Override
  public IBinder onBind(Intent intent) {
    Log.i("DEBUGCAM", "BOUND: " + intent + " with " + intent.getFlags());

    return null;
  }
}

File: app/src/main/java/com/google/android/apps/photos/cameraassistant/CameraAssistantService.kt
package com.google.android.apps.photos.cameraassistant

import android.app.Service
import android.content.Intent
import android.os.IBinder
import android.util.Log

class CameraAssistantService : Service() {
    override fun onBind(intent: Intent): IBinder? {
        Log.i("DEBUGCAM", "BOUND: " + intent + " with " + intent.flags)
        return null
    }
}